_posts/2021-06-07-Synchronizing-physical-clocks-among-different-Subnero-modems.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: post
+title: Synchronizing physical clocks among different Subnero modems
+banner : images/banner-pulse-synchronization.jpg
+date:  2021-06-07
+categories: wnc
+thumbnail: images/pulse-thumbnail-modems.jpg
+---
+
+All Subnero modems have a “real-time clock” to keep track of time using a precise quartz crystal. Based on the defined frequency of this quartz, software running on the modem can precisely monitor the internal physical time given by parameter `phy.time` in Subnero modem.
+
+Although this frequency of the crystal quartz is reasonably stable, it is impossible to guarantee that all the crystals of different modems will operate at the same frequency. A crystal oscillator is an electronic circuit that uses the mechanical resonance of a vibrating piezoelectric crystal to create an electrical signal with the desired frequency. The resonant frequency depends on size, shape, elasticity, and the speed of sound in the material. Due to manufacturing tolerances, these properties are not identical across manufactured crystals, and so different crystals designed for the same nominal frequency produce slightly different frequency signals. Furthermore, as the operating temperature of the crystal changes, its material properties change, and so does its resonant frequency. These differences in frequency are tiny, but over long periods, the differences accumulate and cause the clocks to drift.
```

Comment:
suggestion: "at _exactly_ the same frequency"

Reply:
Fixed.